Dockerfile

```diff
@@ -18,7 +18,6 @@ RUN pip install wheel
 RUN pip install nltk
 RUN pip install -r /ores/requirements.txt
 RUN pip install -r /ores/test-requirements.txt
-RUN pip install codecov pytest-cov
```

Comment:
We don't need to install this anymore?

Reply:
It's in test-requirements.txt and installed above.